### DISCUSSION TITLE
No se valido la entrega del TPI3

### DISCUSSION THREAD

**Discussion by belencaponelli** [dds-jv/foro#302] [Consultas de herramientas] [answered]
Hola! Como estan?

Acabo de hacer la entrega del TPI3 con el tag entrega-final pero no se corre la validacion. 

![image](https://user-images.githubusercontent.com/82619083/125175304-90c8e000-e1a1-11eb-8c54-8ba899d10bf0.png)

El tic verde que suele aparecer al entregar el TP no se muestra.

**Comment by marcosmyh** [comment, marked as answer]
Hola Belén, me parece que es por el nombre del tag. Pusiste "entrega final" en lugar de "entrega-final"
